kymatio/frontend/entry.py

```diff
@@ -10,8 +10,12 @@ def __init__(self, *args, **kwargs):
         kwargs.pop('name')
         kwargs.pop('class_name')
 
-        frontend_suffixes = {'torch' : 'Torch', 'numpy' : 'NumPy', 'tensorflow'
-                : 'TensorFlow', 'keras': 'Keras'}
+        frontend_suffixes = {'torch' : 'Torch', 
+                             'numpy' : 'NumPy',
+                             'tensorflow' : 'TensorFlow',
+                             'keras': 'Keras',
+                             'sklearn': 'Transformer'}
```

Comment:
why not `sklearn`?

Reply:
:man_shrugging: I followed @eugenium on this.

Reply:
What's the convention here? @eickenberg

Reply:
OH ok I was confused, it's all good. Sorry, I forgot it was the name of the class in sklearn which is indeed Transformer. Sorry